tree/ntuple/v7/src/RNTupleMerger.cxx

```diff
@@ -960,9 +947,14 @@ ROOT::RResult<void> RNTupleMerger::Merge(std::span<RPageSource *> sources, const
       }
    }
 
-   RNTupleMergeData mergeData{sources, *fDestination, mergeOpts};
+   // we should have a model if and only if the destination is initialized.
+   if (!!fModel != fDestination->IsInitialized()) {
+      return R__FAIL("Passing an already-initialized destination to RNTupleMerger (i.e. trying to do incremental "
+                     "merging) can only be done if you provided a valid RNTupleModel");
```

Comment:
```suggestion
      return R__FAIL("passing an already-initialized destination to RNTupleMerger (i.e. trying to do incremental "
                     "merging) can only be done with a valid RNTupleModel");
```

Reply:
I prefer the longer version because it might not be immediately clear where this "valid RNTupleModel" should be provided. In fact, I think I want to make it even more obvious, like "if you provided a valid RNTupleModel when constructing the RNTupleMerger" (because it's a bit of a long-distance relation, where you might have constructed the merger far away in the code from the place where you call Merge. I think it's helpful to make the user's life easier by being very verbose with this error)

Reply:
My suggestion mainly is a stylistic one, stemming from the fact that all of the other error messages (as far as I'm aware) are written in the passive voice and non-capitalized. I agree with the verbosity! I would personally prefer to try to keep consistent with the style of phrasing in the errors, but also open to have my mind changed.

Reply:
I understand your point and agree with it to an extent: I'm certainly fixing the non-capitalization, but I wouldn't know how to write the error in a way that doesn't sound weird in a passive voice - and I don't believe it would add value in this case. I'd argue that for user-facing messages it's more important to be easy to read and act upon, rather than keeping a super-consistent formal style (and passive style sometimes hinders readability imho).
However if you have ideas on how to rephrase this message I'm open for suggestions!